### ISSUE TITLE
64 bit adafruit blinka

### ISSUE THREAD

**Issue by wipfli** [wipfli/buildroot#3]
```
Copying src/Adafruit_Blinka.egg-info to /scratch/wipfli/buildroot/output/target/usr/lib/python3.8/site-packages/Adafruit_Blinka-5.2.3-py3.8.egg-info
running install_scripts
ERROR: architecture for "/usr/lib/python3.8/site-packages/adafruit_blinka/microcontroller/bcm283x/pulseio/libgpiod_pulsein" is "ARM", should be "AArch64"
package/pkg-generic.mk:359: recipe for target '/scratch/wipfli/buildroot/output/build/python-adafruit-blinka-5.2.3/.stamp_installed' failed
make[1]: *** [/scratch/wipfli/buildroot/output/build/python-adafruit-blinka-5.2.3/.stamp_installed] Error 1
Makefile:84: recipe for target '_all' failed
make: *** [_all] Error 2
```
I installed the package when I was still using 32 bit linux. Now I switched to 64 bit and I get this error during compiling. If I rerun ```make``` the error does not appear again and it builds everything.

**Comment by wipfli** [comment]
Exclude adafruit_blinka from architecture consistency checking by adding in ```python-adafruit-blinka.mk```:
```
PYTHON_ADAFRUIT_BLINKA_BIN_ARCH_EXCLUDE = /usr/lib/python3.8/site-packages/adafruit_blinka
```

**Comment by wipfli** [comment]
Closed with commit 567e930.
